tbi.datacommons.io/portal/gitops.json

```diff
@@ -160,7 +138,7 @@
       }
     ]
   },
-  "requiredCerts": ["privacy_policy"],
+  "requiredCerts": [],
   "featureFlags": {
     "explorer": true,
```

Comment:
maybe we should disable explorer for now since it won't work anyways (no guppy)

Reply:
i think it's fine because no one should use it until data is in anyway